docs/contributing/Building, Debugging, and Testing on Unix.md

```diff
@@ -7,6 +7,7 @@ Particularly for developers who aren't experienced with .NET Core development on
 1. Clone git@github.com:dotnet/roslyn.git
 1. Run `./build.sh --restore`
 1. Run `./build.sh --build`
+2. Don't forget to update libssh!
```

Comment:
Markdown already auto numbers these.

```suggestion
1. Don't forget to update libssh!
```

Reply:
The actual content of the doc change is not important here, just testing that we are able to go through a fast path in CI :smile:

Reply:
Ah I didn't see the branch name "dont-build-docs-changes" 😄 
Hope things work well 🎉